### PR TITLE
Add get occurences tool

### PR DESCRIPTION
**Pull request by Alanamosse**
Simple wrapper for a Rscript using the spocc package. 
Includes :
 - 1 Rscript
 - 1 wrapper
 - 1 shed file
 - 1 test file

### CODE REVIEW

File: tools/spocc/spocc_occ.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            '$sname' 
+            '$from' 
+            $limit; 
+
+        #if $from=='gbif':
+            sed -z -i -e 's/\n  \"/ /g' -e 's/\n}/ /g' output.tab
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="sname" type="text" label="Scientific name" value="Genus species"/>
```
Comment (bgruening) [inline tools/spocc/spocc_occ.xml]: maybe give an exampe?

File: tools/spocc/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: spocc_occ
+owner: ecology
+categories:
+    -Ecology
```
Comment (bgruening) [inline tools/spocc/.shed.yml]: `- Ecology`

File: tools/spocc/spocc_occ.xml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        <param name="from" type="select" multiple="true" label="Data source to get data from" help="Any combination of gbif, bison, inat, ebird, antweb, ala, idigbio, obis, ecoengine and/or vertnet" value="gbif,bison,inat">
+            <option value="gbif">gbif</option>
```
Comment (bgruening) [inline tools/spocc/spocc_occ.xml]: Do you want to add some more info to these data sources?

File: tools/spocc/spocc_occ.xml (hunk 1 above)
Comment (bgruening) [reply tools/spocc/spocc_occ.xml]: value should be empty right, but `help="e.g. Canis lupus" label="Scientific name of the genus species"`

Comment (Alanamosse) [reply tools/spocc/spocc_occ.xml]: Yes I can improve this. 

Genus species is the format of a scientific name so `label="Scientific name of the genus species"` is kind of redundant. 
We should use smthing like `label="Scientific name of the species" help="Genus species format, eg : Canis lupus"`

File: tools/spocc/.shed.yml (hunk 2 above)
Comment (Alanamosse) [reply tools/spocc/.shed.yml]: Ok thks

File: tools/spocc/spocc_occ.xml (hunk 3 above)
Comment (Alanamosse) [reply tools/spocc/spocc_occ.xml]: Yes good point, I should give the full names of databases here and add links in the help section

**Comment by bgruening** [comment]
Great!!

**Comment by Alanamosse** [comment]
Thank you again @bgruening !

**Comment by bgruening** [comment]
If you want to have this in usegalaxy.eu, please do not forget to add it to https://github.com/usegalaxy-eu/usegalaxy-eu-tools

**Comment by yvanlebras** [comment]
and the initial "Galaxy-E flavour" tool list can help to create it https://github.com/65MO/GalaxyE_docker_galaxy-stable/blob/master/GalaxyE_tool_list.yml

**Comment by yvanlebras** [comment]
PR made for tools list https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/188

**Pull request merged**